ts/WoltLabSuite/Core/Ui/Pagination/JumpToPage.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Handles the "jump to page" function in paginations.
+ *
+ * @author Marcel Werk
+ * @copyright 2001-2022 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module WoltLabSuite/Core/Ui/Pagination/JumpToPage
+ * @since 6.0
+ */
+
+import { dialogFactory } from "../../Component/Dialog";
+import { wheneverFirstSeen } from "../../Helper/Selector";
+import { getPhrase } from "../../Language";
+
+function jumpToPage(element: WoltlabCorePaginationElement): void {
+  const html = `
+    <dl>
+      <dt></dt>
+      <dd>
+        <input
+          class="tiny"
+          type="number"
+          min="1"
+          max="${element.count}"
+          value="${element.count}"
+          aria-label="${getPhrase("wcf.page.jumpTo.pageNo")}"
+          placeholder="${getPhrase("wcf.page.jumpTo.pageNo")}"
```

Comment:
This placeholder is not really seen, because there is a value by default and because the `<input>` is `tiny`. Should this be put as the `<dt>` content instead?